### PR TITLE
Replace parsepatch with rs_parsepatch

### PR DESCRIPTION
**Pull request by calixteman**

### CODE REVIEW

File: bugbug/repository.py [hunk 1]
```diff
@@ -17,7 +17,7 @@
 import hglib
 import requests
 from dateutil.relativedelta import relativedelta
-from parsepatch.patch import Patch
+import rs_parsepatch as rs_pp
```
Comment (marco-c) [inline bugbug/repository.py]: Nit: I'd prefer a more explanatory name, e.g. leave it as `rs_parsepatch`, or `parsepatch`

File: bugbug/repository.py [hunk 2]
```diff
@@ -216,15 +219,14 @@ def _transform(commit):
...
-            else:
-                raise
-
-        sizes.append(after.count(b"\n"))
+        if not stats["deleted"] and not stats["binary"]:
+            try:
+                after = HG.cat([path.encode("utf-8")], rev=commit.node)
+                sizes.append(after.count(b"\n"))
+            except hglib.error.CommandError as e:
+                print((e, stats))
```
Comment (marco-c) [inline bugbug/repository.py]: What error can we have here?

File: bugbug/repository.py [hunk 3]
```diff
@@ -216,15 +219,14 @@ def _transform(commit):
...
-        except hglib.error.CommandError as e:
-            if b"no such file in rev" in e.err:
-                after = b""
-            else:
-                raise
-
-        sizes.append(after.count(b"\n"))
+        if not stats["deleted"] and not stats["binary"]:
+            try:
+                after = HG.cat([path.encode("utf-8")], rev=commit.node)
```
Comment (marco-c) [inline bugbug/repository.py]: Doesn't the rev= need to be bytes?

File: bugbug/repository.py (hunk 2 above)
Comment (calixteman) [reply bugbug/repository.py]: If a file has been deleted but rs_pp didn't see it then we'll catch it here

Comment (marco-c) [reply bugbug/repository.py]: OKOK, can we remove the print statement?

**Pull request merged**